sites/feedback/assets/page-feedback-en.html

```diff
@@ -42,7 +42,7 @@ <h3 class="wb-inv">Give feedback about this page</h3>
 				</fieldset>
 				<div class="gc-pft-no nojs-show">
 					<p id="gc-pft-why" class="nojs-show mrgn-tp-lg mrgn-bttm-md">If not, tell us why below:</p>
-					<p class="nojs-hide wb-inv" aria-live="polite">Tell us why below:</p>
+					<p class="nojs-hide wb-inv" role="status">Tell us why below:</p>
```

Comment:
This is not a status message according to WCAG definition. That don't represent the result of an user action.

The line 80 is the status message and it is marked by using the `role=status`

Reply:
I previously explained my rationale in-depth in https://github.com/wet-boew/GCWeb/pull/2401#issuecomment-2243855184 and https://github.com/wet-boew/GCWeb/pull/2401#issuecomment-2256146925.

But I'll explain exactly why I think the no message fits within ARIA and WCAG's definitions below... :).

---

**Here's [ARIA 1.2 spec's definition of the ``status`` role](https://www.w3.org/TR/wai-aria-1.2/#status):**
> **``status`` role**
> A type of [live region](https://www.w3.org/TR/wai-aria-1.2/#dfn-live-region) whose content is **advisory information** for the user but is not important enough to justify an [alert](https://www.w3.org/TR/wai-aria-1.2/#alert), often but not necessarily presented as a status bar.

IMO "Tell us why below:" is advisory information that isn't important-enough to warrant an alert nor a status bar. It's triggered as the direct result of a user action (i.e. pressing the "No" button).

---

**Here's [WCAG 2.x's definition of a status message](https://www.w3.org/WAI/WCAG22/Understanding/status-messages.html#dfn-status-message):**
> **status message**
> change in content that is not a [change of context](https://www.w3.org/WAI/WCAG22/Understanding/status-messages.html#dfn-changes-of-context), and that provides information to the user on the success or **results of an action**, on the waiting state of an application, on the **progress of a process**, or on the existence of errors

IMO "Tell us why below:" represents both information on the result of the act of pressing the "No" button, as well as the progress of the process of providing feedback. It's the second "step" in the process of providing negative feedback.